plugins/formal-verification/formver.core/src/org/jetbrains/kotlin/formver/conversion/ContractDescriptionConversionVisitor.kt

```diff
@@ -171,15 +171,18 @@ class ContractDescriptionConversionVisitor(
     private fun embeddedVarByIndex(ix: Int): VariableEmbedding =
         if (ix == -1) signature.receiver!! else signature.params[ix]
 
-    private fun VariableEmbedding.nullCmp(isNegated: Boolean, source: KtSourceElement? = null): ExpEmbedding =
+    private fun VariableEmbedding.nullCmp(
+        isNegated: Boolean,
+        sourceRole: SourceRole = SourceRole.Unknown,
```

Comment:
Is the default value ever used here?

Reply:
No, this has been changed.

Reply:
Is it?  There seems to still be a default value in the newest code.